libs/feature/editor/src/lib/+state/editor.effects.ts

```diff
@@ -16,10 +20,18 @@ export class EditorEffects {
   saveRecord$ = createEffect(() =>
     this.actions$.pipe(
       ofType(EditorActions.saveRecord),
-      withLatestFrom(this.store.select(selectRecord)),
-      switchMap(([, record]) =>
-        this.editorService.saveRecord(record).pipe(
-          map(() => EditorActions.saveRecordSuccess()),
+      withLatestFrom(
+        this.store.select(selectRecord),
+        this.store.select(selectRecordFieldsConfig)
+      ),
+      switchMap(([, record, fieldsConfig]) =>
+        this.editorService.saveRecord(record, fieldsConfig).pipe(
+          switchMap((newRecord) =>
+            of(
+              EditorActions.saveRecordSuccess(),
+              EditorActions.openRecord({ record: newRecord })
```

Comment:
A bit overkill opening what you just saved, no? Or there might be some changes made by the backend?